File: 1° Período - Introdução a Programação 1/Exercicios em Python/Lista 3 - Python/q8.py
#Crie um algoritmo que receba nome, sexo (M ou F) e idade de uma pessoa. Se a pessoa for do sexo
#feminino e tiver menos do que 25 anos, imprimir seu nome e a mensagem: “Aceita!”. Caso contrário, imprimir o nome
#e a mensagem: “Não aceita!”.

nome=input("Insira o nome da pessoa: ")
sexo=input("Insira M(para masculino) e F(para feminino: ")
idade=int(input("Insira a idade de uma pessoa: "))
if sexo=="F" or sexo=="f" and idade<25:
    print(nome, "ACEITA!")
else:
    print(nome, "NÃO ACEITA!")

File: 1° Período - Introdução a Programação 1/Exercicios em C/Lista 3 - C/q8.c
//Crie um algoritmo que receba nome, sexo (M ou F) e idade de uma pessoa. Se a pessoa for do sexo
//feminino e tiver menos do que 25 anos, imprimir seu nome e a mensagem: “Aceita!”. Caso contrário, imprimir o nome
//e a mensagem: “Não aceita!”.

#include<stdio.h>
int main(){
	char sexo;
	char nome[10];
	int idade;
	printf("Nome: ");
	scanf("%s", &nome);
	printf("Insira M(para masculino) e F(para feminino: ");
	scanf("%s", &sexo);
	printf("Insira a idade: ");
	scanf("%d", &idade);
	if(sexo=="F" && idade<25)
	{
		printf("ACEITA!");
	}   
	else
	{
		printf("NÃO ACEITA!");
	}	    
}
